pip/commands/uninstall.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import absolute_import
 
+import pip
```

Comment:
This should import like ``from pip.index import FormatControl`` or ``import pip.index``.